### PR TITLE
change UI

### PR DESCRIPTION
**Pull request by MinhDo2007**

### CODE REVIEW

File: app/views/shared/_user_info.html.erb [hunk 1]
```diff
@@ -1,2 +0,0 @@
-<%= link_to gravatar_for(current_user, size: {with: Settings.setting.size_avatar}), current_user %>
-<h1><%= current_user.name %></h1>
```
Comment (phamkykhoi) [inline app/views/shared/_user_info.html.erb]: Checkout this file

File: app/views/shared/_search.html.erb [hunk 2]
```diff
@@ -1,27 +1,29 @@
...
-          <%= radio_button_tag :filter_state, item, 
-            checked = (params[:filter_state] == item || params[:filter_state].blank?) %> 
-          <%= item.titleize %>
-        <% end %>
-      <% end %>
+    <div class="col-md-4">
+      <%= text_field_tag :search, params[:search], class: "form-control" %>
+    </div>
+    <div class="col-md-1">
+        <%= submit_tag t("search"), class: "form-control" %>
```
Comment (NguyenTanDuc) [inline app/views/shared/_search.html.erb]: indent

File: app/views/shared/_search.html.erb [hunk 3]
```diff
@@ -1,27 +1,29 @@
...
     <div id="search-field">
-      <div class="col-md-5">
+      <div class="col-md-4">
         <%= select_tag "category_id", options_from_collection_for_select(@categories, 
-          "id", "name", params[:category_id]), include_blank: t("word.all") %>
-        <%= text_field_tag :search, params[:search] %>
-      <%= submit_tag t("search"), class: "btn btn-primary btn-default" %>
+          "id", "name", params[:category_id]), include_blank: t("word.all"), class: "form-control" %>
+        <div class="tm-radio">
+        <% [Settings.setting.all, Settings.setting.learned, Settings.setting.not_learned].each do |item| %>
```
Comment (NguyenTanDuc) [inline app/views/shared/_search.html.erb]: indent

File: app/views/shared/_user.html.erb [hunk 4]
```diff
@@ -1,2 +1,11 @@
-<%= link_to gravatar_for(current_user, size: {with: Settings.setting.size_avatar}), current_user %>
-<h1><%= current_user.name %></h1>
+<section class="user_info">
+  <h1>
+    <%= gravatar_for @user, size: Settings.gravatar.large_size %>
+    <%= @user.name %>
+  </h1>
+  <h5>
+    <%= t "learned" %> 
+    <%= (@lessons.lesson_learned(@user).count) * Settings.question_number %>
```
Comment (NguyenTanDuc) [inline app/views/shared/_user.html.erb]: remove ()

File: app/views/words/index.html.erb [hunk 5]
```diff
@@ -7,4 +7,5 @@
     </tr>
     <%= render @words %>
   </table>
+    <%= will_paginate @words %>
```
Comment (NguyenTanDuc) [inline app/views/words/index.html.erb]: indent

File: app/views/shared/_search.html.erb [hunk 6]
```diff
@@ -1,27 +1,29 @@
...
+      <%= text_field_tag :search, params[:search], class: "form-control" %>
+    </div>
+    <div class="col-md-1">
+        <%= submit_tag t("search"), class: "form-control" %>
     </div>
-    <div class="tm-submit">
-      <%= f.submit t("Filter"), class: "btn btn-primary btn-default" %>
-      <%= link_to t("CSV"), words_path(format: "csv"), class: "btn btn-primary btn-default" %>
+    <div class="col-md-1">
+    <%= link_to t("csv"), words_path(format: "csv"), class: "form-control" %>
```
Comment (khanhkhanh) [inline app/views/shared/_search.html.erb]: Indentation

**Pull request merged**